microcosm_flask/swagger/schemas.py

```diff
@@ -65,9 +65,15 @@ def iter_fields(self, schema: Schema) -> Iterable[Tuple[str, Field]]:
         Generates: name, field pairs
 
         """
+        serialized_names = {}
+        for field_name, field_def in schema.declared_fields.items():
+            if isinstance(field_def, Field) and field_def.data_key:
```

Comment:
What cases is this `isinstance(field_def, Field)` supposed to guard against?

Reply:
I'm guessing this is mostly a belts-and-suspenders type deal.  I can remove it, it's not adding clarity.